src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random randomNumber = new Random();
+
+    public static Random getRandomNumber() {
+        return randomNumber;
+    }
+
+    public static void setRandomNumber(Random randomNumber) {
+        Lottery.randomNumber = randomNumber;
+    }
+
+    public static Ball getRandomBall() {
+        Ball rundomBall;
+        rundomBall = new Ball(ColorSupplier.getRandomColor(), randomNumber.nextInt(100));
```

Comment:
avoid using magic number

Reply:
Could you please explain what you mean? The only number here is 100; what is so magic about it? What do you want to change?